### PR TITLE
fix(all-services): fix the pending sonar issues in all services

### LINKED ISSUE #2013: All services - Sonar fix
**Describe the bug**
Fix all the sonar issues raised 
https://sonarcloud.io/summary/new_code?id=sourcefuse_loopback4-microservice-catalog

### PR DESCRIPTION
**Pull request by arpit1503khanna**
## Description
Fix the pending sonar issues in all services

Fixes #2013 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/cli/src/generators/cdk/index.ts [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-  readFile,
-  writeFile,
   appendFile,
+  readFile,
   readdir,
   rename,
+  writeFile,
 } from 'node:fs/promises';
-import fetch from 'node-fetch';
+import zlib from 'node:zlib';
```
Comment (yeshamavani) [inline packages/cli/src/generators/cdk/index.ts:10]: ??

File: sandbox/cache-example/public/index.html
Comment (yeshamavani) [inline sandbox/cache-example/public/index.html:1]: ??

File: packages/cli/src/generators/cdk/index.ts (hunk 1 above)
Comment (arpit1503khanna) [reply packages/cli/src/generators/cdk/index.ts:10]: After lint fix the order of imports changed

File: sandbox/cache-example/public/index.html
Comment (arpit1503khanna) [reply sandbox/cache-example/public/index.html:1]: This is also happened lint:fix

**Pull request merged**